bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -53,13 +53,18 @@
         <div class="inner-container">
           <h2>{{ _('Making Virtual Reality a reality for all') }}</h2>
           <p>{{ _('Using A-Frame, developers, designers and artists are able to easily create accessible VR experiences.') }}</p>
-          <a rel="external" href="https://mozvr.com/#showcase" class="button">{{ _('Try it out') }}</a>
+          <a rel="external" href="https://aframe.io/" class="button">{{ _('Explore A-Frame') }}</a>
         </div>
       </div>
       <footer>
         <ul>
+        {% if l10n_has_tag('emerging_tech_links') %}
+          <li><a rel="external" href="https://research.mozilla.org/mixed-reality/">{{ _('Learn about WebVR') }}</a></li>
+          <li><a rel="external" href="https://aframe.io/blog/">{{ _('Visit A-Frame blog') }}</a></li>
+        {% else %}
           <li><a rel="external" href="https://aframe.io/">{{ _('Explore A-Frame') }}</a></li>
```

Comment:
Could maybe have this link read "Learn more" and point to `https://research.mozilla.org/mixed-reality/`? As is, it's repeating the link on the button-ish CTA above.